src/icon/lib/icon.scss

```
@@ -0,0 +1,18 @@
+.tbds-icon {
+  --size: 1em;
+
+  fill: currentcolor;
```

May not be an issue, as CSS tends to be so forgiving, but I believe this value is camel case, per the spec:

```suggestion
  fill: currentColor;
```